config/networking/istio-authorization-policies.yml

```diff
@@ -0,0 +1,37 @@
+#@ load("@ytt:data","data")
+
+#@ def principal(namespace, service_account):
+#@   return "cluster.local/ns/{}/sa/{}".format(namespace, service_account)
+#@ end
+
+#! https://istio.io/latest/docs/reference/config/security/authorization-policy/
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
```

Comment:
@cloudfoundry/cf-networking 

What do y'all think about us introducing this `AuthorizationPolicy` use case?

Reply:
seems ok